### PR TITLE
BLUEBUTTON-1644: Enable JVM option required for flamegraphs

### PR DESCRIPTION
**Pull request by karlmdavis**
Per https://medium.com/netflix-techblog/java-in-flames-e763b3d32166, this is needed to generate CPU flamegraphs.

Also note this quote about the option: "We’ve found in tests that this costs between 0 and 3% extra CPU, depending on the workload."

That seems an acceptable cost to me, given the usefulness of flame graphs for performance profiling.

https://jira.cms.gov/browse/BLUEBUTTON-1644

### CODE REVIEW

File: ops/ansible/playbooks-ccs/build_bfd-server.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@
         data_server_dir: "{{ bfd_server_dir }}"
         data_server_container: 'will_not_be_used'  # The JBoss instance used here is already (manually) installed.
         data_server_container_name: jboss-eap-7.0
-        data_server_appserver_jvmargs: "-Xmx{{ (((data_server_ec2_instance_type_mem_mib | int) * 0.80) - 2048) | int }}m -XX:MaxMetaspaceSize=2048m -XX:MaxMetaspaceSize=2048m -Xloggc:{{ bfd_server_dir }}/gc.log -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintTenuringDistribution"
+        data_server_appserver_jvmargs: "-Xmx{{ (((data_server_ec2_instance_type_mem_mib | int) * 0.80) - 2048) | int }}m -XX:MaxMetaspaceSize=2048m -XX:MaxMetaspaceSize=2048m -Xloggc:{{ bfd_server_dir }}/gc.log -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintTenuringDistribution -XX:+PreserveFramePointer"
```
Comment (njdister) [inline ops/ansible/playbooks-ccs/build_bfd-server.yml]: Totally unrelated to your work, but we seem to be setting `-XX:MaxMetaspaceSize=2048m` twice.  Any particular reason or can that be removed?

File: ops/ansible/playbooks-ccs/launch_bfd-server.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@
       vars:
         data_server_dir: "{{ bfd_server_dir }}"
         data_server_appserver_https_port: "{{ vault_data_server_appserver_https_port }}"
-        data_server_appserver_jvmargs: "-Xmx{{ (((data_server_ec2_instance_type_mem_mib | int) * 0.80) - 2048) | int }}m -XX:MaxMetaspaceSize=2048m -XX:MaxMetaspaceSize=2048m -Xloggc:{{ bfd_server_dir }}/gc.log -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintTenuringDistribution"
+        data_server_appserver_jvmargs: "-Xmx{{ (((data_server_ec2_instance_type_mem_mib | int) * 0.80) - 2048) | int }}m -XX:MaxMetaspaceSize=2048m -XX:MaxMetaspaceSize=2048m -Xloggc:{{ bfd_server_dir }}/gc.log -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintTenuringDistribution -XX:+PreserveFramePointer"
```
Comment (njdister) [inline ops/ansible/playbooks-ccs/launch_bfd-server.yml]: Same as above.

File: ops/ansible/playbooks-ccs/build_bfd-server.yml (hunk 1 above)
Comment (karlmdavis) [reply ops/ansible/playbooks-ccs/build_bfd-server.yml]: No clue why that's there. Could you submit a small PR to fix it up?

File: ops/ansible/playbooks-ccs/launch_bfd-server.yml (hunk 2 above)
Comment (karlmdavis) [reply ops/ansible/playbooks-ccs/launch_bfd-server.yml]: No clue why that's there. Could you submit a small PR to fix it up?

**Pull request merged**